data-model.sql

```diff
@@ -341,62 +341,47 @@ CREATE TABLE alternate_care_facility_address_line (
 		MATCH FULL ON DELETE CASCADE ON UPDATE CASCADE		
 );
 		
+CREATE TABLE name_type (
+	id bigserial not null,
+	code varchar(1),
+	description varchar(100),
+	CONSTRAINT name_type_pk PRIMARY KEY (id)
+);
+ALTER TABLE name_type OWNER TO pulse;
+
+CREATE TABLE name_representation (
+	id bigserial not null,
+	code varchar(1),
+	description varchar(100),
+	CONSTRAINT name_representation_pk PRIMARY KEY (id)
+);
+ALTER TABLE name_representation OWNER TO pulse;
+
+CREATE TABLE name_assembly (
+	id bigserial not null,
+	code varchar(1),
+	description varchar(100),
+	CONSTRAINT name_assembly_pk PRIMARY KEY (id)
+);
+ALTER TABLE name_assembly OWNER TO pulse;
+
 CREATE TABLE patient (
 	id bigserial not null,
-	given_name varchar(100) not null,
-	family_name varchar(100) not null,
+	full_name varchar(255) not null,
+	friendly_name varchar(128) not null,
 	dob date,
 	ssn varchar(15),
 	gender varchar(10),
-	phone_number varchar(100),
-	address_id bigint,
 	alternate_care_facility_id bigint,
```

Comment:
should address_id be removed at this point? i know it will be removed shortly, just not sure why it is related to the name change.